features/serializable-errors.yml

```diff
@@ -0,0 +1,13 @@
+name: Serializable errors
+description: "`DOMException`, `Error`, `EvalError`, `RangeError`, `ReferenceError`, `SyntaxError`, `TypeError`, and `URIError` are serializable (structuring cloning is possible). This allows you to pass errors from a worker using `postMessage()`."
+group: javascript
```

Comment:
I'm not sure this is a JavaScript feature. `structuredClone()` isn't defined by ECMAScript, right? I know this doesn't exactly matter, since we're talking about features in a browser context, but I'd hate to contribute to some confusion on what is a JS thing versus a web API thing.

```suggestion
```

Reply:
Yeah, I agree this should rather not be in the javascript group